test/span.js

```diff
@@ -182,6 +183,39 @@ describe('span should', () => {
         assert.isOk(unnormalizedKey in Span._getBaggageHeaderCache());
     });
 
+    describe('post trace sampling tests for span', () => {
```

Comment:
might want to rename because it's not really post trace sampling and when we finally do implement post trace sampling, this test name might get confusing